decidim-core/spec/lib/form_builder_spec.rb

```diff
@@ -762,7 +762,7 @@ def organization
           filename:,
           attached?: present?,
           attachment: double(
-            blob:,
+            :blob => blob,
             "image?" => image?
```

Comment:
This can be converted to symbol,
```suggestion
            image?: => image?
```

Reply:
There's an error in your syntax, as you're using the colon and also the arrow. I applied what I understand you meant in 17245eb573 😄

Reply:
Now it's on 984ace2000ad97e768bb8eba091d4ba7eda2ac11 - I had to force push because I missed a change